CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add possibility for a Toolbar to rearrange its items according to space available @miroslavstastny ([#1657](https://github.com/stardust-ui/react/pull/1657))
 - Edit `buttonBehavior` Adding aria-disabled when button has loading state @kolaps33 ([#1789](https://github.com/stardust-ui/react/pull/1789))
 - Added `audio-off` and `clipboard-copied-to` icon to Teams theme @bcalvery ([#1792](https://github.com/stardust-ui/react/pull/1792))
+- Added `window-maximize`, `window-minimize` and `window-unmaximize` icons to Teams theme @codepretty ([#1803](https://github.com/stardust-ui/react/pull/1803))
```

Comment:
What about `window-restore`? As in FontAwesome:

![image](https://user-images.githubusercontent.com/14183168/63158596-e3330d80-c019-11e9-9580-2230e841db5f.png)

Reply:
Good find! I didn't notice those, so wasn't sure what to use for "unmaximize". Will update that to `window-restore`